app/javascript/components/widgets/widgets/land-cover/tree-cover/actions.js

```diff
@@ -1,41 +1,54 @@
+import axios from 'axios';
 import { getExtent } from 'services/forest-data';
 
 export const getData = ({ params, dispatch, setWidgetData, widget }) => {
-  getExtent(params)
-    .then(response => {
-      const extent = response.data && response.data.data;
-      let totalArea = 0;
-      let cover = 0;
-      let plantations = 0;
-      let data = {};
-      if (extent.length) {
-        totalArea = extent[0].total_area;
-        cover = extent[0].value;
-        data = {
-          totalArea,
-          cover,
-          plantations
-        };
-      }
-      if (params.indicator) {
-        dispatch(setWidgetData({ data, widget }));
-      } else {
-        getExtent({ ...params, forestType: 'plantations' }).then(
-          plantationsResponse => {
-            const plantationsData =
-              plantationsResponse.data && plantationsResponse.data.data;
-            plantations = plantationsData.length ? plantationsData[0].value : 0;
-            if (extent.length) {
-              data = {
-                ...data,
-                plantations
-              };
+  axios
+    .all([
+      getExtent(params),
+      getExtent({ ...params, forestType: null, landCategory: null })
+    ])
+    .then(
+      axios.spread((response, adminResponse) => {
+        const extent = response.data && response.data.data;
+        const adminExtent = adminResponse.data && adminResponse.data.data;
+        let totalArea = 0;
+        let totalCover = 0;
+        let cover = 0;
+        let plantations = 0;
+        let data = {};
+        if (extent.length) {
+          totalArea = adminExtent[0].total_area;
```

Comment:
You need to check admin response is defined here and below too otherwise it might crash the app.